components/schemes/README.md

```diff
@@ -10,119 +10,195 @@ path: /catalog/theming/
 
-For example, there is a scheme for both the Material Design typography system and the Material
-Design color system. Most components have a themer for at least one of these systems.
+The Container scheme represents a single configurable entity that is applicable to all themeable components. A container scheme consists of all of the subsystem schemes.
+
+Theming extensions are component extensions that, when invoked with a default container scheme, will theme a component according to the [Material Design guidelines](https://material.io/design). When provided with subsystem schemes via a container scheme, the extension will map the subsystem scheme's values to the component’s parameters.
 
 ### Sensible defaults, yet highly configurable
 
-By default, an instance of a scheme is initialized with the Material defaults. You can use these
-defaults as a baseline, but at a minimum we encourage you to tweak your color scheme's primary and
-secondary colors to match your brand colors.
+By default, components have reasonable defaults for all of their customizable properties, e.g. `backgroundColor` or `titleFont`. You can use these defaults as a baseline, but we encourage you to theme your components to match your brand style using theming extensions.
 
 ### Schemes
 
 <ul class="icon-list">
-  <li class="icon-list-item icon-list-item--link"><a href="Color/">Color scheme</a></li>
-  <li class="icon-list-item icon-list-item--link"><a href="Shape/">Shape scheme</a></li>
-  <li class="icon-list-item icon-list-item--link"><a href="Typography/">Typography scheme</a></li>
+<li class="icon-list-item icon-list-item--link"><a href="Container/">Container scheme</a></li>
+<li class="icon-list-item icon-list-item--link"><a href="Color/">Color scheme</a></li>
+<li class="icon-list-item icon-list-item--link"><a href="Shape/">Shape scheme</a></li>
+<li class="icon-list-item icon-list-item--link"><a href="Typography/">Typography scheme</a></li>
 </ul>
 
 ## Examples
 
-### Creating a scheme
+### Theme component
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
 
-In order to access the scheme APIs you'll first need to add the scheme target to your Podfile:
+```swift
+import MaterialComponents.MaterialButtons
+import MaterialComponentsBeta.MaterialButtons_Theming
+import MaterialComponentsBeta.MaterialContainerScheme
+
+let containerScheme = MDCContainerScheme()
+let button = MDCButton()
+button.applyTextTheme(withScheme: containerScheme)
+```
+
+#### Objective-C
+
+```objc
+#import <MaterialComponents/MaterialButtons.h>
+#import <MaterialComponents/MaterialButtons+Theming.h>
+#import <MaterialComponentsBeta/MaterialContainerScheme.h>
 
-```bash
-pod 'MaterialComponents/schemes/Color'
-pod 'MaterialComponents/schemes/Shape'
```

Comment:
Where is the CocoaPods instruction moving to?

Reply:
I have moved this to the lowest portion of each respective section as @jverkoey request from pervious comment.